### DISCUSSION TITLE
Withdrawal issue

### DISCUSSION THREAD

**Discussion by encoderrrr** [blockscout/blockscout#10371] [Support]
I withdrew 8 TIA from Forma yesterday, but it still hasn't appeared in my Keplr wallet. Could you please help me? Thank you.

https://explorer.forma.art/tx/0x2bc01f69b047095d31fb9edda7e92794caeb9c2f2c96fa901a0d1a3bcb0f497c?tab=index

**Comment by ekawibisono** [comment]
Hello @encoderrrr this is blockscout github, please ask to team forma art. thanks
